<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1314</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>821</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>基本信息</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>821</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>毕业情况</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>290</y>
     <width>295</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="addButtom">
      <property name="text">
       <string>添加</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="updateButton">
      <property name="text">
       <string>修改</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="deleteButton">
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>164</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>专业班级</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="majority">
      <item>
       <property name="text">
        <string>软件1801</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>软件1802</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>软件1803</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>通信1801</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>通信1802</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>220</y>
     <width>350</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>就业意向</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QCheckBox" name="cppPro">
        <property name="text">
         <string>CPP开发</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="javaPro">
        <property name="text">
         <string>JAVA开发</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="webPro">
        <property name="text">
         <string>前端开发</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>220</y>
     <width>190</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>毕业意向</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="postExam">
      <property name="text">
       <string>考研</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="findJob">
      <property name="text">
       <string>就业</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="infoTable">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>350</y>
     <width>1071</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>314</x>
     <y>70</y>
     <width>745</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>学号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="stuID"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>姓名</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="stuName"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>性别</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="man">
      <property name="text">
       <string>男</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="woman">
      <property name="text">
       <string>女</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>入学年份</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="timeIntoCollege"/>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>stuID</tabstop>
  <tabstop>stuName</tabstop>
  <tabstop>man</tabstop>
  <tabstop>woman</tabstop>
  <tabstop>majority</tabstop>
  <tabstop>postExam</tabstop>
  <tabstop>findJob</tabstop>
  <tabstop>cppPro</tabstop>
  <tabstop>javaPro</tabstop>
  <tabstop>webPro</tabstop>
  <tabstop>addButtom</tabstop>
  <tabstop>updateButton</tabstop>
  <tabstop>deleteButton</tabstop>
  <tabstop>infoTable</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
